Dockerfile.manifest

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.13.15-alpine3.12
+FROM golang:1.14.6-alpine3.12
```

Comment:
Upstream is building with either 1.13 or 1.15; do we feel safe using 1.14?

Reply:
https://github.com/kubernetes/kubernetes/commit/aa2bb18a2b1c8d2f42bda82f0377f6415dbc6944